tests/test_operation.py

```diff
@@ -171,6 +174,18 @@ def test_operation_init(self, test_class, monkeypatch):
         if n == 0:
             return
 
+    def test_qubit_unitary_init(self):
+        """Test for the init of QubitUnitary"""
+        wires = [0]
+        U = qml.RX(0.3, wires=0).matrix
+
+        op = qml.QubitUnitary(U, wires=wires)
+
+        assert op.name == qml.QubitUnitary.__name__
+        assert np.allclose([U], op.data)
```

Comment:
This line and the next line are maybe redundant?

Reply:
😆